### PR TITLE
Catalog-My-Things

### PR DESCRIPTION
**Pull request by vanelnw**
In this project we :

- Insert and query data in SQL.
- Use primary key & foreign key mechanisms for joining tables.
- Understand the different types of relationships between tables.
- Query multiple tables.
- Prepare complex queries that answer analytical questions.
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Apply Ruby best practices and language style guides in code.
- Describe the SOLID principles of OOP.
- Implement classes and objects in Ruby.
- Understand the four main principles of OOP.
- Implement encapsulation and inheritance with Ruby.
- Create a UML class diagram.
- Explain the difference between associations, aggregations, and composition in OOP.
- Set up associations between classes and objects.
- Store data in files.
- Build interactive console apps.
- Recognize the value of making equal contributions to group projects to produce the best outcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,141 @@
...
+
+```
+  ruby solver.rb
+```
+
+test 
+
+```
+  rspec solver.rb
+```
```
Comment (elisha2kyakpo1) [inline README.md:76]: - [ ] I suggest you put up the right file for starting the app, `ruby main.rb`. Running `solver` file will not start the app.

File: README.md [hunk 2]
```diff
@@ -1 +1,141 @@
...
+- Twitter: [@yousef](https://twitter.com/Yousef45653478)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] 
+
```
Comment (elisha2kyakpo1) [inline README.md:107]: - [ ] Let's not leave these sections empty, if you do not have anything to write, just write `N/A` or `Coming soon`

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ All the tests are passing👏
👍 Descriptive commit messages.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- You can add success or error messages to communicate with users when an item is added or did not add.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED ✔️ 
Hi @yosefHesham, @vanelnw and @momotsow, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Alt Text
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**